awx_collection/plugins/modules/tower_application.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+
+# (c) 2017, Wayne Witzel III <wayne@riotousliving.com>
```

Comment:
This copyright line should be changed to this year + applicable author name/email

Reply:
resolved in update